scanner/updater/export.go

```diff
@@ -62,18 +62,23 @@ func Export(ctx context.Context, outputDir string) error {
 	outOfTree = append(outOfTree, updaterSet.Updaters())
 
 	for i, uSet := range [][]string{
-		{"oracle", "photon", "suse", "aws", "rhcc"},
-		{"alpine", "rhel", "ubuntu", "osv", "debian"},
+		{"oracle", "aws", "rhcc"},
+		{"alpine", "rhel", "debian"},
+		{"ubuntu", "suse", "photon"},
+		{"osv"},
```

Comment:
Question: My guess is OSV takes more memory to run. Would you happen to know why?

Question: Are other categories grouped by resource consumption to avoid workflow failures or is it arbitrary?

Reply:
I checked last ten failures, Most failures happen in either OSV or Ubuntu. So I decided not to group them in one updater.

Reply:
is it possible to add more memory to our CI instances?

Reply:
I had a discussion with Gavin quite a while ago and he mentioned that "I'm pretty sure (95%) that our GHA runners cannot be resized because we rely on opensource repo budget. So you are stuck with figuring a way to reduce the memory footprint." https://redhat-internal.slack.com/archives/CC5UHD0KA/p1684780027001869?thread_ts=1684523068.748449&cid=CC5UHD0KA

Reply:
> I checked last ten failures, overwhelmingly most failures happen in either OSV or Ubuntu. So I decided not to group them in one updater.

Why don't we perform one updater for each feed?

Also, it's important to note that the order of the updaters is significant and some updaters are performed in isolation. There is no indication anywhere in the code, right?

Reply:
> > I checked last ten failures, overwhelmingly most failures happen in either OSV or Ubuntu. So I decided not to group them in one updater.
> 
> Why don't we perform one updater for each feed?
> 
> Also, it's important to note that the order of the updaters is significant and some updaters are performed in isolation. There is no indication anywhere in the code, right?

That's a good suggestion. But at the same time, grouping Photon, SUSE, etc is working just fine. So I think one data source for each updater is a bit overkill.